test/tests/functional/pbs_multi_sched.py

```diff
@@ -647,6 +647,268 @@ def test_multi_fairshare(self):
         self.assertEquals(n.nshares, sc3_shares)
         self.assertEquals(n.usage, sc3_usage)
 
+    @timeout(300)
+    def test_fairshare_usage(self):
+        """
+        Test the schedulers fairshare usage file and
+        check the usage file is updating correctly or not
+        """
+        self.common_setup()
+        rv = self.server.add_resource('foo', 'float', 'nh')
+        self.assertTrue(rv)
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {'job_history_enable': 'True'}, expect=True)
```

Comment:
This is not required in the test case.
As we discussed remove this. Use log match to verify that the jobs has finished.

Reply:
I agree with @varunsonkar. No need to use the custom resource "foo" here.